CorsixTH/Lua/dialogs/resizables/file_browsers/load_map.lua

```diff
@@ -28,7 +28,8 @@ function UILoadMap:UILoadMap(ui, mode)
   local path = ui.app.user_level_dir
   local treenode = FilteredFileTreeNode(path, ".map")
   treenode.label = "Maps"
-  self:UIFileBrowser(ui, mode, _S.load_game_window.caption:format(".map"), 295, treenode, true)
```

Comment:
`load_game_window` name for loading a map?

Save uses `save_map_window`, so that seems ok.

Reply:
Ah, in #2593. Fair enough.